app/models/pbs/event/course.rb

```diff
@@ -13,6 +13,8 @@ module Pbs::Event::Course
   APPROVALS = %w(requires_approval_abteilung requires_approval_region
                  requires_approval_kantonalverband requires_approval_bund).freeze
 
+  COURSE_NUMBER_REGEX = /\APBS CH(?: [A-Z]{2})? [0-9]{3}-[0-9]{2}\z/.freeze
```

Comment:
Following what's specified in https://pilot.scouts.ch (page 12 in the current version)